CHANGELOG.md

```diff
@@ -32,6 +32,13 @@ added to your `.bazelrc` file.
 build --@io_opentelemetry_cpp//api:with_abseil
 ```
 
+### New Libraries
+
+The following experimental libraries are now available:
+
+- [Cloud BigQuery Control API](/google/cloud/bigquerycontrol/README.md) -
+This library provides support for the Dataset, Job, Model, Project, Routine, RowAccessPolicy, and Table REST resources.
```

Comment:
nit: can you align this to 80 characters?

We turned off `mdformat`-ing the changelog because I didn't like how some things were rendered in GH's release notes.